src/Tickets/Commerce/Gateways/PayPal/Assets.php

```diff
@@ -94,7 +94,7 @@ public function register() {
 				'tribe-tickets-commerce-base-gateway-checkout-js',
 				'tribe-tickets-commerce-base-gateway-checkout-toggler',
 			],
-			null,
+			'wp_enqueue_scripts',
```

Comment:
This will end up loading the asset all the time when should_enqueue_assets() is true. Do we want to add this to the checkout group? When is it not loading that we need it loaded?